applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_embedded_solver.py

```diff
@@ -108,7 +108,7 @@ def ImportModelPart(self):
         ## Construct the Trilinos import model part utility
         self.trilinos_model_part_importer = trilinos_import_model_part_utility.TrilinosImportModelPartUtility(self.main_model_part, self.settings)
         ## Execute the Metis partitioning and reading
-        self.trilinos_model_part_importer.ExecutePartitioningAndReading()
+        self.trilinos_model_part_importer.ImportModelPart()
```

Comment:
IMO ExecutePartitioningAndReading is a much more self-explicative name of what this method does.

Reply:
I changed it bcs now the utility can also do restarts (and possibly also other imports, think of hdf5,...)
This way whenever a new functionality is being added in this utility it will be also available for the solvers.

Then the name `ExecutePartitioningAndReading` does not fit any more

Reply:
Go ahead then!

Reply:
@philbucher are we sure we want to add all the new functionalities to the same utility? I'd rather have separate classes to read mdpa, hdf5 and restart files, for example.

Reply:
@jcotela this is what I would like to avoid, otherwise we will have a lot of copy-pasting with more options becoming available

Reply:
@philbucher I agree in principle, but my preferred solution to that would be a good I/O base class all these utilities can derive from... I guess we will see what works when we get there though...

Reply:
I talked to @jcotela and we agreed that we leave the proposed solution and change it in the future to e.g. a factory